src/Symfony/Bundle/FrameworkBundle/Command/AbstractConfigCommand.php

```diff
@@ -64,6 +65,11 @@ protected function findExtension(string $name)
         $bundles = $this->initializeBundles();
         $minScore = INF;
 
+        $kernel = $this->getApplication()->getKernel();
+        if ($kernel instanceof ExtensionInterface && $kernel instanceof ConfigurationExtensionInterface && $name === $kernel->getAlias()) {
```

Comment:
I get that this is needed for `ConfigDumpReferenceCommand`
but why is the `instanceof ConfigurationExtensionInterface` check required in this base class
Put another: why don't we have a similar check for bundles?
Or do we miss a check in `ConfigDumpReferenceCommand` for bundles too?

Reply:
Note that in the linked issue, the kernel doesn't implement `ConfigurationExtensionInterface` apparently.

Reply:
Ok, i've updated my code, so now it checks for `ConfigurationInterface` OR `ConfigurationExtensionInterface`.